src/Microsoft.Azure.SignalR.Management/RestApiProvider.cs

```diff
@@ -16,27 +16,26 @@ internal class RestApiProvider
 
         private readonly RestApiAccessTokenGenerator _restApiAccessTokenGenerator;
 
-        private readonly string _audienceBaseUrl;
-        private readonly string _serverEndpoint;
+        private readonly ServiceEndpoint _serviceEndpoint;
+
+        private string AudienceBaseUrl => _serviceEndpoint.AudienceBaseUrl;
```

Comment:
No need to use property for private member. We can make it readonly and assign the value in the constructor.